Bottom: f630f326ef99cba10e6aae9c014618e9ede076a1
Top:    88bba080aac2fb2cc0db59233df27aba25bac2f8
Author: Jason Wang <jasowang@redhat.com>
Date:   2018-05-14 14:49:50 +0800

Refresh of vhost_net-passing-raw-xdp-buff

---

diff --git a/drivers/vhost/net.c b/drivers/vhost/net.c
index aff338a..80add75 100644
--- a/drivers/vhost/net.c
+++ b/drivers/vhost/net.c
@@ -637,7 +637,7 @@ static void handle_tx_copy(struct vhost_net *net)
 		vq->heads[nheads].id = cpu_to_vhost32(vq, head);
 		vq->heads[nheads].len = 0;
 
-		err = vhost_net_build_xdp(nvq, &msg.msg_iter, &xdp);
+		err = vhost_net_build_xdp(nvq, sock, &msg.msg_iter, &xdp);
 		if (!err)
 			msg.msg_control = &xdp;
 		else
